dragonfly_doe2/doe/floor_space.py

```diff
@@ -0,0 +1,149 @@
+from dragonfly.room2d import Room2D
+from dragonfly.story import Story
+from dataclasses import dataclass
+from typing import List
+
+
+@dataclass
+class Wall:
+    """*.inp Wall object.
+    Example:
+        .. code-block::f#
+            "simple_example_dfb_Floor1_Room1 Wall_1" = EXTERIOR-WALL
+               CONSTRUCTION    = "EWall Construction"
+               LOCATION        = SPACE-V1
+               ..
+    """
+    name: str = None
+    location_index: int = None
+    wall_con_name: str = None
+
+    @classmethod
+    def from_room_seg(cls, _name: str, _bc_id: int, _wall_con_name: str):
```

Comment:
Here are another habits of yours that I have to change! 😄

1. No `_` unless you have a good reason for it.
2. No short/cryptic names unless you really have to.
3. No redundant object name in the argument name.

```suggestion
    def from_room_segment(cls, name: str, boundary_condition_id: int, construction: str):
```

Reply:
I appreciate the opertunity for identification and rectification of bad habits!